client/my-sites/current-site/notice.jsx

```diff
@@ -199,25 +197,6 @@ export class SiteNotice extends React.Component {
 		);
 	}
 
-	freeToPaidPlanNotice() {
```

Comment:
I haven't thoroughly inspected why, but migrating this to the newly-introduced JITM template has broken the domain with plan nudge click email campaign from triggering, because `clickUpgradeNudge()` action is not dispatched.

Could we address this need? I think this is a very good scenario to cover to prove if the new template can be general enough :)

Reply:
> I haven't thoroughly inspected why, but migrating this to the newly-introduced JITM template has broken the domain with plan nudge click email campaign from triggering, because clickUpgradeNudge() action is not dispatched.

Sorry, I wasn't aware of the email campaign. Where can I find the information about it?

Reply:
You can find the details in p2-p9jf6J-28v. tl;dr from the technical side, you can just open up the network inspector, click on the nudge, and see if there is a request for the nudge click endpoint. The test plan of https://github.com/Automattic/wp-calypso/pull/36495 would be a good reference here :)

Reply:
Thanks @southp ! Although the PR already supports custom actions, the marketing related middle ware was not automatically loaded.